packages/app/src/debug/DebugContainer.vue

```diff
@@ -54,13 +69,26 @@ fragment DebugSpecs on Query {
       __typename
       ... on CloudProject {
         id
-        runByNumber(runNumber: 11) {
+        runByNumber(runNumber: 3) {
```

Comment:
Any particular significance to `3`, or just making a bit easier to test locally pending a "real" value here?

Reply:
I didn't mean to include this in the changes - was just switching back and forth between different runs when doing local testing. Reverted